app/models/peoplefinder/token.rb

```diff
@@ -10,6 +11,8 @@ class Peoplefinder::Token < ActiveRecord::Base
 
   validate :valid_email_address
 
+  DEFAULT_TTL = 3
```

Comment:
My default assumption for a TTL is that it's going to be in seconds. Perhaps we should include `_HOURS`/`_hours` in the names.